src/content/config.ts

```diff
@@ -0,0 +1,18 @@
+import { z, defineCollection } from 'astro:content';
```

Comment:
🚨 Typescript! 😱

Astro actually uses Typescript by default. We've opted out as much as possible but we opted to use this file as it validates the frontmatter of our guides at build time. This ensures that any guides added will always have great metadata + SEO open graph tags 💪

Reply:
Nice! 🔥